### PR TITLE
Adds Some More Images to !Mentor and !Yoggers

### PR DESCRIPTION
**Pull request by UselessTheremin**
froggy left, and nobody else seemed like they were gonna do it. so here we go. there are a few lingering images to coder that i didnt add and may at a later date but im tired rn

i sure hope im doing this right

....
am i?

### CODE REVIEW

**Comment by AshCorr** [comment]
Good stuff

File: models/Discord/Commands/DiscordCommandMentor.js [hunk 1]
```diff
@@ -162,6 +162,13 @@ class DiscordCommandMentor extends DiscordCommand {
 		"https://cdn.discordapp.com/attachments/734475284446707753/774177356637536266/unknown.png",
 		"https://cdn.discordapp.com/attachments/734475284446707753/774496437307572264/unknown.png",
 		"https://cdn.discordapp.com/attachments/734475284446707753/775422450379784202/Sin_titulo.png"
+	    	//March Update
+	    	"https://media.discordapp.net/attachments/734475284446707753/826626004704559154/image0.png"
```
Comment (AshCorr) [inline models/Discord/Commands/DiscordCommandMentor.js:166]: Missing commas after each line, this is whats causing the build to fail.

File: models/Discord/Commands/DiscordCommandYoggers.js [hunk 2]
```diff
@@ -47,6 +47,10 @@ class DiscordCommandYoggers extends DiscordCommand {
 	"https://cdn.discordapp.com/attachments/734475284446707753/768725068393611296/unknown.png",
 	"https://cdn.discordapp.com/attachments/734475284446707753/770375580855697428/unknown.png",
 	"https://cdn.discordapp.com/attachments/134722036353204224/722263699527565312/coggers.png"
+	"https://media.discordapp.net/attachments/734475284446707753/825033507868639262/image0.png"
```
Comment (AshCorr) [inline models/Discord/Commands/DiscordCommandYoggers.js:50]: same as above

**Review by AshCorr — CHANGES_REQUESTED** [review]

**Comment by UselessTheremin** [comment]
@AshCorr fixed

**Pull request merged**